# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1747221247 -7200
# Parent  fd4247a0d37546fd480e901c900093f59c66c7dd
NNN23 - Follow-up to bug 1956292: Move 'Always underline links' setting where it belongs.

diff --git a/mail/components/preferences/general.inc.xhtml b/mail/components/preferences/general.inc.xhtml
--- a/mail/components/preferences/general.inc.xhtml
+++ b/mail/components/preferences/general.inc.xhtml
@@ -325,8 +325,13 @@
                     font-incoming-email-label.value,
                     default-font-reply-checkbox.label"/>
         </hbox>
       </hbox>
+      <separator class="thin"/>
+      <hbox align="center">
+        <checkbox id="alwaysUnderlineLinks" data-l10n-id="always-underline-links"
+                  preference="layout.css.always_underline_links"/>
+      </hbox>
     </html:fieldset>
     </html:div>
 
     <!-- Date and time formatting -->
@@ -581,14 +586,8 @@
 
     <html:div data-category="paneGeneral">
     <html:fieldset data-category="paneGeneral">
       <vbox>
-        <hbox align="center">
-          <checkbox id="alwaysUnderlineLinks"
-                    data-l10n-id="always-underline-links"
-                    preference="layout.css.always_underline_links"/>
-        </hbox>
-
         <hbox>
           <checkbox id="viewAttachmentsInline"
                     data-l10n-id="view-attachments-inline"
                     preference="mail.inline_attachments"/>
